### PR TITLE
Copy edit El Paso notebook for clarity

### PR DESCRIPTION
**Pull request by steveryb**
This doesn't get it to a point where I think it would work for a public
tutorial (will open a future PR for that), but helps make
it clearer.

Tried to add some copy editing here to:

- correct some errors (repeating a phrase, listing a footnote where
there wasn't one, etc)
- add clarity (rephrased some things that I thought were confusing,
tried to improve explanations if there were low hanging fruit)

### CODE REVIEW

**Comment by stuhlmueller** [comment]
Thank you Steven! :+1:

@brachbach review and merge?

File: notebooks/src/el-paso-workflow.ipynb [hunk 1]
```diff
@@ -194,14 +194,9 @@
...
-    "Why sample instead of manipulating distributions directly?\n",
+    "Why sample instead of manipulating distributions directly? To use distributions in our prediction, we need to be able to combine distributions for sub-predictions together. Combining distributions can be difficult, as it requires non-trivial math. \n",
     "\n",
-    "As I build out my model, we'll see that: \n",
-    "\n",
-    "- Composing samplers is easy - it's just programming as usual\n",
-    "- Composing probability distributions is hard - it's math\n",
-    "\n",
-    "I'll build up complex models by composing samplers."
+    "Ergo allows you to create samplers (like the `ergo.lognormal_from_interval` above), which you can compose together using regular python operators (`*, +, -`). This allows you to focus on how you're modeling the problem, rather than the math. I'll build a complex model for this problem by composising samplers together."
```
[the remark this answers is not in the document]
Comment (steveryb) [reply notebooks/src/el-paso-workflow.ipynb]: Agh, great catch. Need to figure out how to spell check notebooks for next time. Fixed!

**Comment by steveryb** [comment]
I accidentally made this from my fork, lmk @brachbach if you want me to resubmit from a branch in the ergo repo.

**Comment by brachbach** [comment]
@steveryb thanks a bunch for this PR!

FYI we're crunching on something else this week, so I likely won't get to review this until next week.

**Comment by steveryb** [comment]
No worries! I'm going to add a preamble too this then, as I think that helps with someone coming across it, but not urgent at all. Good luck with the crunch :)

**Comment by steveryb** [comment]
This this is on my fork, and it doesn't look like there's an easy way to change the branch a PR is based on, I'm going to close this and redo it with an ought fork.

**Comment by steveryb** [comment]
Made https://github.com/oughtinc/ergo/pull/326.

**Pull request closed without merging**